batch2/Dockerfile.worker

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:18.04
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && \
+  apt-get -y install \
+    python3 python3-pip && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
So I think we can do way better than this by using `python:3.6-slim-stretch`.  Stretch is Debian 9 which I think is pretty close to Ubuntu 18.  It has python3 and pip(3) already installed.  That image is only 48MB: https://hub.docker.com/layers/python/library/python/3.6-slim-stretch/images/sha256-3f74562f0a4667c2a97f90b7513f8a874da6e9f62ee8e6d195c678bf18f368a5